test/xavierTests.dproj

```diff
@@ -0,0 +1,557 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@nunopicado I believe that we don't need this Delphi project group in this repository. This file could give us more work to maintain and it is not required to make things work. Do you agree?

Reply:
The project group file is very useful to keep things clean and simple in the Delphi IDE, and it is just a single file that rarely needs a change, if any, so the cost is very low.

It's not, however, a hard task to create the file every time a fork is made, adding just a little more work to open both projects (xavier and test) and saving them as a group.

I will keep it in my end, since it makes sense for two related projects to be linked, but if you don't think it is necessary, it can be safely removed from git tracking.

Use you best judgement, I'll support your call.

Reply:
So, no problem. Keep the file.